tools/commit_slider/utils/helpers.py

```diff
@@ -137,25 +137,52 @@ def getBlobDiff(file1, file2):
         content = file.readlines()
     with open(file2) as sampleFile:
         sampleContent = sampleFile.readlines()
-    # ignore first line with memory address
-    i = -1
     curMaxDiff = 0
-    for sampleLine in sampleContent:
-        i = i + 1
+    for i, sampleLine in enumerate(sampleContent):
+        if i == 0:
+            # ignore first line with memory address
+            continue
         if i >= len(sampleContent):
             break
         line = content[i]
+        if "nan" in sampleLine.lower() or "nan" in line.lower():
+            # todo: test value
+            return 1000
+            import sys
+            return sys.float_info.max
```

Comment:
1. Do we need to resolve this TODO? It seems like the second "return" is unreachable
2. Do I understand correctly that the difference is reported even if the both `sampleLine` and `line` have NaNs? Why do we need such behavior? I'd say if both outputs have nans at the same places, the lines are equal, and the big difference shouldn't be reported

Reply:
1. todo removed
2. I agree. It's better to handle correctly

Reply:
2. We still return `bigFloatNumber` even if both values are NaN:
https://github.com/openvinotoolkit/openvino/blob/866bd1ea008a483e444e92dc11329d87beffbb49/tools/commit_slider/utils/helpers.py#L149-L150

Can we return `bigFloatNumber` only when only one of the strings has NaN?

Reply:
Done